terraform/production/s3.tf

```diff
@@ -32,6 +32,19 @@ resource "aws_s3_bucket_policy" "protected_binaries_restricted" {
         "Principal": "*",
         "Action": "s3:GetObject",
         "Resource": "arn:aws:s3:::${each.key}/*"
+      },
+      {
+        "Sid": "StesachsAtAWSWriteSubFolder",
+        "Effect": "Allow",
+        "Principal": {
+          "AWS": "arn:aws:iam::679174810898:root"
```

Comment:
I am not able to verify this, where does this ARN come from?

Reply:
@stephenmsachs sent me this ARN for himself.

Reply:
Then that is probably fine then. If he can just verify that is indeed correct here then I think this looks okay to me.

Reply:
Double checked, it's the correct arn.